src/wildcard-dns-http-server/main.go

```diff
@@ -100,7 +101,8 @@ func dnsServer(conn *net.UDPConn, group *sync.WaitGroup) {
 			continue
 		}
 		var txtAnswers = []dnsmessage.Resource{}
-		for _, txt := range txts {
+		var answer string = ""
```

Comment:
строка пустая по дефолту
```suggestion
		var answer string
```